src/Microsoft.ML.AutoML/SweepableEstimator/Estimator.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.SearchSpace;
+using Microsoft.ML.AutoML.CodeGen;
+
+namespace Microsoft.ML.AutoML
+{
+    /// <summary>
+    /// abstract class for estimator.
+    /// </summary>
+    public class Estimator
```

Comment:
You say its abstract in the comment but the class is not actually defined as abstract.

Reply:
And is the user meant to use this class? If not it shoudln't be public.

Reply:
NVM, if the user needs to use the inherited classes it needs to be public, but if it is meant to be abstract we should still declare it that way.

Reply:
### Resolved